svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/[issueSearchType=issue_search_type]/+page.server.ts

```diff
@@ -46,7 +47,35 @@ const buildNavigationFilterOptions = <TParameter extends string>(
   });
 };
 
-export const load: PageServerLoad = async ({ fetch, params, url: { searchParams, href } }) => {
+const redirectIfRequired = ({
```

Comment:
If you were asked to describe what this function does, in a paragraph (or 2) what would you say?

Reply:
I almost think you are trolling me. 👀 

What I mean is, under what condition should this redirect actually happen?